### PR TITLE
Add clang-format and bazel build to github actions.

### PR DESCRIPTION
**Pull request by nastarann**
Hi Andrey,

In this PR I added clang-format and bazel build test to the CI.

### CODE REVIEW

**Comment by a-litvinov** [comment]
check-format is failing for this cl, can you look into this?

**Comment by nastarann** [comment]
All errors fixed. PTAL.

File: BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier")
+
+# Bazel buildifier: auto formats bazel files.
+buildifier(
```
Comment (a-litvinov) [inline BUILD.bazel:18]: I wonder why buildifier is defined in BUILD.bazel, while the other go tools (that I assume buildifier depends on) are in WORKSPACE?

Comment (nastarann) [reply BUILD.bazel:18]: That's right. buildifier dependencies added to WORKSPACE, but it also needs a build rule in root BUILD file. It complains if there is no BUILD file at root dir.

**Pull request merged**